style.css

```diff
@@ -40,4 +40,125 @@ button {
 
 .textError {
     color: red;
+}
+
+.repo-card svg {
+    width: 15px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Repo-card classes */
+
+.repo-card {
+    border: 1px solid var(--card-border);
+    border-radius: 0.3em;
+    background-color: transparent;
+    color: var(--card-text);
+    padding: 1em;
+}
+
+.repo-link {
+    color: var(--card-text);
+    font-size: 1.1em;
+}
+
+.repo-card a:hover {
+    text-decoration: underline;
+}
+
+.repo-details-style {
+    font-size: 0.8em;
+}
+
+.repo-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    list-style: none;
+    padding: 0;
+}
+
+/* Languages */
+
+.lng {
```

Comment:
I would rename this. `lng` is language, but I don't know what is referring to.

Maybe rename to `lng-badge`